test/web/10-api.js

```diff
@@ -34,7 +34,9 @@ describe('Profile Manager API', () => {
       try {
         result = await profileManager.createProfile({
           type: 'Person',
-          content: {name: 'Mike Smith', color: '#000000'}
+          content: {name: 'Mike Smith', color: '#000000'},
```

Comment:
This bit was in the unit tests, but I'm not sure `profileManager.createProfile()` supports those params anymore (type and content).  Should these be removed?

Reply:
@dmitrizagidulin just reviewed, and yes, `content` and `type` there are no longer used.

Reply:
Np, I'll remove em from the test, while I'm here.